addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -221,10 +221,13 @@ const Link = Widget.extend({
      * @private
      */
     _correctLink: function (url) {
+        url = url.replace(/\s/g, '');
         if (url.indexOf('mailto:') === 0 || url.indexOf('tel:') === 0) {
             url = url.replace(/^tel:([0-9]+)$/, 'tel://$1');
         } else if (url.indexOf('@') !== -1 && url.indexOf(':') === -1) {
             url = 'mailto:' + url;
+        } else if (/^[^a-zA-Z]*[0-9]+[^a-z]*$/.test(url)) {
+            url = 'tel://' + url;
```

Comment:
What is this `//`? Is that a wrong copy paste of the `http://` or it has a meaning?

Reply:
I did it to be consistent with what is done two lines above. If a user type tel:04 95 88 65 32 in the link URL, we adapt it with tel://, so I figured I'd better do the same.. What do you think ?

Reply:
Indeed, it is not only about ios but chrome on linux too, it will redirect to domain.com/0499468715 somehow.. while with // it will work (launch the popup to transfer the phone/start the call on my phone)